src/main/java/seedu/address/logic/parser/ImportCommandPreparer.java

```diff
@@ -0,0 +1,124 @@
+//@@author chantca95
+package seedu.address.logic.parser;
+
+import javafx.stage.FileChooser;
+import javafx.stage.Stage;
+import seedu.address.logic.commands.ImportCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.person.Address;
+import seedu.address.model.person.Email;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Phone;
+import seedu.address.model.tag.Tag;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Optional;
+import java.util.Set;
+
+public class ImportCommandPreparer {
+    ArrayList<Person> persons = new ArrayList<>();
+    
```

Comment:
Create enum, just a simple
``` 
public static enum Attributes { 
    NAME, PHONE, EMAIL, ADDRESS, TAG
 }
```
should be enough, then replace all the attributes[1] with attributes[PHONE] etc.